response_operations_ui/controllers/edit_contact_details_controller.py

```diff
@@ -14,12 +14,22 @@ def edit_contact_details(edit_details_data, respondent_id):
     url = f'{app.config["BACKSTAGE_API_URL"]}/v1/party/update-respondent-details/{respondent_id}'
 
     response = requests.put(url, json=edit_details_data)
-    if response.status_code != 200:
-        logger.debug('Error changing contact details', respondent_id=respondent_id)
-        return False
-
-    logger.debug('Successfully changed contact details', respondent_id=respondent_id)
-    return True
+    if response.status_code == 200:
+        logger.debug('Respondent details updated', respondent_id=respondent_id, status_code=response.status_code)
+        return True, ''
+
+    if response.status_code == 409:
+        logger.debug('Email address already exists', respondent_id=respondent_id, status_code=response.status_code)
```

Comment:
This information could be usefully so I would change it to info, rather then debug